From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 17 Nov 2020 15:35:15 +0100
Subject: js/shellDbus: Do not forward device IDs

We already pass the device event node path for identification purposes,
we can stop doing that here.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1501>
---
 js/ui/shellDBus.js | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/js/ui/shellDBus.js b/js/ui/shellDBus.js
index 75a2f34..bac9caa 100644
--- a/js/ui/shellDBus.js
+++ b/js/ui/shellDBus.js
@@ -151,9 +151,10 @@ var GnomeShell = class {
 
         let connection = this._dbusImpl.get_connection();
         let info = this._dbusImpl.get_info();
-        let params = { 'device-id': GLib.Variant.new('u', device.get_device_id()),
-                       'timestamp': GLib.Variant.new('u', timestamp),
-                       'action-mode': GLib.Variant.new('u', Main.actionMode) };
+        let params = {
+            'timestamp': GLib.Variant.new('u', timestamp),
+            'action-mode': GLib.Variant.new('u', Main.actionMode),
+        };
 
         let deviceNode = device.get_device_node();
         if (deviceNode)
